src/index.html

```diff
@@ -7,6 +7,406 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <div class="header__logo">
+        <a href="#Home">
+          <img
+            src="./images/logo.png"
+            alt="logo"
+            class="header__img"
+          >
+        </a>
+      </div>
+
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a href="#apple" class="nav__link is-active"> Apple </a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#samsung" class="nav__link">Samsung</a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#smartphones" class="nav__link">Smartphones</a>
+          </li>
+
+          <li class="nav__item">
+            <a
+              href="#laptops&computers"
+              class="nav__link"
+              data-qa="nav-hover"
+            >
+              Laptops & Computers
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#gadgets" class="nav__link">Gadgets</a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#tablets" class="nav__link">Tablets</a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#photo" class="nav__link">Photo</a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#video" class="nav__link">Video</a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+
+    <main class="main">
+      <div class="product main__product">
+        <div class="product__card" data-qa="card">
+          <div class="product__image">
+            <img
+              src="./images/imac.jpeg"
+              alt="iMac"
+              class="product__image product__image--image"
+            >
+          </div>
+
+          <div class="product__container">
+            <div class="product__desc">
+              <h2 class="product__title">
+                APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+              </h2>
+              <p class="product__code">Product code: 195434</p>
+            </div>
```

Comment:
Add empty line between multiline siblings. See more details in checklist, point 2.
```suggestion
                APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
              </h2>
              
              <p class="product__code">Product code: 195434</p>
            </div>
```

Reply:
Also in html file you've made same mistakes as in the previous task. So correct them here also.